create_dmd_release/create_dmd_release.d

```diff
@@ -667,16 +660,23 @@ void createRelease(string branch)
     }
 
     // Copy bin64
-    version(Windows) {} else // Win doesn't include 64-bit tools
+    if(do64Bit)
     {
-        if(do64Bit)
+        copyFile(cloneDir~"/dmd/generated/"~osDirName~"/release/64/dmd"~exe, releaseBin64Dir~"/dmd"~exe);
+        version(Windows)
+        {
+            // patch sc.ini to point to optlink.exe in bin folder
+            auto sc_ini = cast(string)std.file.read(cloneDir~"/dmd/ini/windows/bin/sc.ini");
+            sc_ini = sc_ini.replace(`%@P%\optlink.exe`, `%@P%\..\bin\optlink.exe`);
+            std.file.write(releaseBin64Dir~"/sc.ini", sc_ini);
+        }
+        else // Win doesn't include 64-bit tools
         {
-            copyFile(cloneDir~"/dmd/generated/"~osDirName~"/release/64/dmd"~exe, releaseBin64Dir~"/dmd"~exe);
             copyDir(cloneDir~"/tools/generated/"~osDirName~"/64", releaseBin64Dir, file => !file.endsWith(obj));
             copyFile(cloneDir~"/dub/bin/dub64"~exe, releaseBin64Dir~"/dub"~exe);
-            if (codesign)
-                signBinaries(releaseBin32Dir);
```

Comment:
This would have never signed the 64-bit executables, but there have been none so far. Signing should include DLLs, too, though.